### DISCUSSION TITLE
Support for FaunaDB

### DISCUSSION THREAD

**Discussion by tfadsilva** [prisma/prisma#2407] [Q&A] [answered]
Hey everyone. I apologize for the duplication of issues here, as there is an already open issue on the subject (https://github.com/prisma/prisma/issues/350), but it seems stale and no one has had an official response there in months.

It is mentioned in that issue that support for FaunaDB within Prisma is planned, with a user posting a link to a repository for a Rust Fauna Driver that was being developed to be later used by Prisma. But that repository was created a year ago, had strong activity by a user for two months, and then it died. 

I assume one of the possible reasons for this is the current need for developers focused on working in Prisma 2.0, which is totally understandable. I was just wondering if we could get an official statement on whether this is still a planned feature or if it is no longer part of your plans. 

Thank you and keep up the amazing work on Prisma!!

**Comment by ryands17** [comment, marked as answer]
Hey @evilmonster18 :wave: 

Yes you are correct! Currently we are focusing on making the Prisma Client and Migrations stable first, and then we will be able to take on other databases like Fauna.

Yes it is on our list, but we cannot provide an exact timeline for this :)

**Reply by janpio** [reply]
For the record, this is our issue tracking FaunaDB: https://github.com/prisma/prisma/issues/350

**Comment by tfadsilva** [comment]
Okay @ryands17, thank you for the response. Looking forward to seeing this in the future!

**Comment by ludralph** [comment]
Hi @tfadsilva,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
